src/resources/MachineLearning/UserActionHistoryConfiguration/UserActionHistoryConfiguration.ts

```diff
@@ -1,17 +1,29 @@
 import API from '../../../APICore.js';
 import Resource from '../../Resource.js';
 
+/**
+ * @deprecated
```

Comment:
Looking at swagger I see it's also deprecated on the backend, do we have a date for the removal? I would expect a message in the jsdoc to explain a bit more

Reply:
@GermainBergeron I have some information, Marie told me : 
`The call is still maybe used in hipaa I think, and if it is the case, it would be max until January. So after that, we will check to see if we have people who use it secretly, imo in February max we can remove everything, it is dependent on the "Context API" project.`

Reply:
done